CHANGELOG.md

```diff
@@ -5,11 +5,13 @@
 ### Internal
```

Comment:
"Remove unused code" name sounds really vague. Can we update the commit message to indicate which part of unused code we are removing instead, please?  

For example, "Remove unused section header state usages` or something like that.